en/orm/saving-data.rst

```diff
@@ -874,8 +874,8 @@ the singular, underscored version of the association name. For example::
 Saving HasOne Associations
 --------------------------
 
-When saving hasOne associations, the ORM expects a single nested entity at the
-singular, underscored version of the association name. For example::
+When saving hasOne associations, the ORM expects a single nested entity named with the
+singular, lowercase version of the association name. For example::
```

Comment:
Ah, yes I think that's a great idea.

Now that you put it that way (and I'm sure that it was meant like that) it makes much more sense... in all those phrases the word 'underscored' was implying the Cake's *Inflector::underscore* method which will both lowercase and underscore multi-word arguments.

I guess that hint is more obvious to more seasoned Cake users but it went completely over my n00b head; in that light it totally makes sense to link to Inflector page from there (it would take care of clearing out what is meant).

I'll try and see if I manage to create a table that summarizes all the Inflector's results based on the methods described on that page.